service/generators.gradle

```diff
@@ -0,0 +1,41 @@
+dependencies {
+	implementation 'io.swagger.core.v3:swagger-annotations:2.1.12'
+	runtimeOnly 'org.webjars.npm:swagger-ui-dist:4.1.3'
+	swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.32'
```

Comment:
Shouldn't need a version for annotations or codegen, since it's set with spring dependency management in the root build.gradle